index.html

```diff
@@ -312,7 +312,7 @@ <h4 class="header grey-text text-darken-3">Reader's Favorites</h4>
          <div class="col s2">
         <div class="card">
           <div class="card-image">
-            <a href="bookdetails.html"><img src="https://www.booktopia.com.au/http_coversbooktopiacomau/9780349415796/140xNxsecrets-in-death.jpg.pagespeed.ic.ZI4rgU80p8.jpg" class="responsive-img"></a>
+            <a href="./templates/bookdetails.html"><img src="https://www.booktopia.com.au/http_coversbooktopiacomau/9780349415796/140xNxsecrets-in-death.jpg.pagespeed.ic.ZI4rgU80p8.jpg" class="responsive-img"></a>
```

Comment:
Nice attempt @amarachukwu-agbo. You could have assigned book images to `cards` in a 
smarter way without repeating div tags.